pygmt/src/grd2cpt.py

```diff
@@ -126,14 +127,14 @@ def grd2cpt(grid, **kwargs):
         also :gmt-docs:`cookbook/features.html#manipulating-cpts`.
     output : str
         Optional parameter to set the file name with extension .cpt to store
-        the generated CPT file. If not given or False [Default], saves the CPT
-        as the session current CPT.
+        the generated CPT file. If not given or ``False`` [Default], saves the
+        CPT as the session current CPT.
```

Comment:
```suggestion
        CPT as the current session CPT.
```
In my opinion the words should switched here.

Reply:
I think both "session current CPT" and "current session CPT" are inaccurate. The key point is that, GMT has several hierarchical levels (https://docs.generic-mapping-tools.org/dev/cookbook/features.html#gmt-modern-mode-hierarchical-levels), thus a "current CPT" could be the current CPT of a session/figure/subplot/panel/inset, depending  in which level the `makecpt`/`grd2cpt` module is called.

Reply:
I have made some changes to the docstrings of `grd2cpt` and `makecpt`. @seisman and @michaelgrund do you feel this is an improvement?